README.md

```diff
@@ -18,6 +18,10 @@ scheduling appointments as easy as possible.
 
 ## Quick Start
 
+1. Clone this repository
+1. Run `npm init`
+1. Start the service by `npm run dev`
```

Comment:
Perhaps worth adding a note explaining more thoroughly what this does?

Reply:
What do you have in mind? Maybe, add the note the way you think would be helpful?